### PR TITLE
Fixed the dev bug in generate docs

### PR DESCRIPTION
**Pull request by yaakovpraisler**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/35013

## Description
Fixed issue where **generate-docs** does not add the used scripts.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: demisto_sdk/commands/generate_docs/generate_playbook_doc.py [hunk 1]
```diff
@@ -127,6 +127,8 @@ def get_playbook_dependencies(playbook: dict, playbook_path: str) -> Tuple[list,
                                 integrations.add(integration_dependency)
         else:
             script_name = task.get('scriptName')
+            if not script_name:
```
Comment (roysagi) [inline demisto_sdk/commands/generate_docs/generate_playbook_doc.py:130]: you can do this change in one line - might look better

**Review by roysagi — COMMENTED** [review]
@yaakovpraisler Please add a unit test

File: demisto_sdk/commands/generate_docs/generate_playbook_doc.py (hunk 1 above)
Comment (yaakovpraisler) [reply demisto_sdk/commands/generate_docs/generate_playbook_doc.py]: done.

File: CHANGELOG.md [hunk 2]
```diff
@@ -11,6 +11,7 @@
 * Fixed an issue where running **format** on a playbook whose `name` does not equal its `id` would cause other playbooks who use that playbook as a sub-playbook to fail.
 * Added support for local custom command configuration file `.demisto-sdk-conf`.
 * Updated the **format** command to include an update to the description file of an integration, to remove community/partner details.
+* Fixed issue where **generate-docs** does not add the used scripts to the README file.
```
Comment (roysagi) [inline CHANGELOG.md:14]: please move this line to the current release.
```suggestion
* Fixed an issue where the **generate-docs** command did not add all the used scripts in a playbook to the README file.
```

**Review by roysagi — APPROVED** [review]
Looks good, just fix and move the release notes line.

**Pull request merged**